taverna-robundle/src/test/java/org/apache/taverna/robundle/validator/ValidatorTest.java

```diff
@@ -21,23 +21,45 @@
 
 import static org.junit.Assert.*;
 
+import java.io.IOException;
+import java.nio.file.CopyOption;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
 
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
 public class ValidatorTest {
 
-	private Path path = Paths.get("src/test/resources/workflowrun.bundle.zip");
+	 Path path;
+	    @Before
+	      public void copyExample() {
+	                try {
+						path = Files.createTempFile("test", ".bundle.zip");
+					} catch (IOException e1) {
+						// TODO Auto-generated catch block
+						e1.printStackTrace();
```

Comment:
If this fails (e.g. out of disk space?), then a printStackTrace does not help as much as simply letting the error make the whole test fail by cahnging `copyExample` to have `throws IOEXception`